src/Stormpath/Authc/Api/ApiAuthenticationResult.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Stormpath\Authc\Api;
+
+
+class ApiAuthenticationResult extends AuthenticatorResult
+{
+
+}
```

Comment:
On the spec I only see that a "result" is expected, not necessarily  a different result type per authenticator type. I think as a general good practice, if a class has no implementation at all, it's very likely that it is not needed; in the case of this implementation there are multiple classes that are unnecessary (all the ones that extend `AuthenticatorResult`).

I think we still have time to remove these classes because per the spec on the README, people are just expecting a "result" which could still be an `AuthenticatorResult`.